### PR TITLE
Centralize info on participation

### PR DESCRIPTION
**Pull request by nlharris**
Currently distributed among menus or linked from other pages.

Not sure whether < li > will work in the web page, and I can't check that in the preview, so I'll have to wait for someone to merge this PR and then fix the formatting if needed.

### CODE REVIEW

File: docs/participate.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: doc
+id: Participate
+title: How to participate in the OBO Foundry
+---
+
+The OBO Foundry is currently a volunteer-run organization. There are many ways to get involved:  
+ <ul>
```
Comment (cmungall) [inline docs/participate.md:8]: in general, markdown is preferred over HTML as more maintainable, but we can do as a separate PR

**Review by cmungall — APPROVED** [review]

File: docs/participate.md (hunk 1 above)
Comment (nlharris) [reply docs/participate.md:8]: ok, i was being lazy; i copied it from a .html page and was hoping not to have to hand-edit it into md. but i can if that is preferred!

Comment (nlharris) [reply docs/participate.md]: @cmungall i converted it to .md

**Review by cmungall — APPROVED** [review]

**Review by matentzn — APPROVED** [review]
